share/pvss_test.go

```diff
@@ -0,0 +1,198 @@
+package share
+
+import (
+	"testing"
+
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/edwards"
+	"github.com/dedis/crypto/random"
+)
+
+func TestPVSS(t *testing.T) {
+
+	suite := edwards.NewAES128SHA256Ed25519(false)
+
+	G := suite.Point().Base()
+	H, _ := suite.Point().Pick(nil, suite.Cipher([]byte("H")))
+
+	n := 20
```

Comment:
`10` or even lower should be good enough, and it'll make the test go way faster. It's currently 1.2, 1.3 sec for this test only on my machine. Same for `TestPVSSDelete`.